actionview/lib/action_view/helpers/form_options_helper.rb

```diff
@@ -137,24 +137,24 @@ module FormOptionsHelper
       #
       # ==== Gotcha
       #
-      # The HTML specification says when +multiple+ parameter passed to select and all options got deselected
-      # web browsers do not send any value to server. Unfortunately this introduces a gotcha:
-      # if a +User+ model has many +roles+ and have +role_ids+ accessor, and in the form that edits roles of the user
-      # the user deselects all roles from +role_ids+ multiple select box, no +role_ids+ parameter is sent. So,
+      # The HTML specification says when the +multiple+ parameter is passed to select and all options were deselected,
```

Comment:
While we are making changes here, `parameter` might be confused with request parameters. Mozilla also [uses specified as the verb](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/select#attributes).
```suggestion
      # The HTML specification says when the +multiple+ attribute is specified on a select element and all options are deselected,
```

Reply:
@p8 great suggestion. Updated!